python_modules/dagster/dagster/_core/definitions/assets.py

```diff
@@ -254,18 +257,25 @@ def __init__(
             backfill_policy, "backfill_policy", BackfillPolicy
         )
 
-        if selected_asset_keys is None:
```

Comment:
context to this change for Nick and Sandy: currently for multi assets we just filter asset checks down to the ones that target selected_asset_keys. That's still the default because AssetSelection includes checks for assets by default, but it will be passed in over selected_asset_check_handles now

Reply:
`it will be passed in over selected_asset_check_handles now`

What precisely do you mean by this?

Reply:
ben is going to start sending explicit check selections from the front end, instead of defaulting to None